generators/ci-cd/templates/_Jenkinsfile

```diff
@@ -156,6 +156,27 @@ node {
 <%_ if (jenkinsIntegrations.includes('docker')) { _%>
 <%= gitLabIndent %>    }
 <%_ } _%>
+<%_ if (jenkinsIntegrations.includes('publishDocker')) { _%>
+
+<%= gitLabIndent %>    def dockerImage
+<%= gitLabIndent %>    stage('build docker') {
+    <%_ if (buildTool === 'maven') { _%>
+<%= gitLabIndent %>        sh "cp -R src/main/docker target/"
```

Comment:
For me, both solutions are correct. So I'm ok for both, as with Gradle, we don't use a plugin

For Maven, see the doc https://github.com/spotify/docker-maven-plugin#usage
we can use: `./mvnw docker:build -DpushImage` too, but I don't know how we will manage the credentials